libnavigation-core/api/current.txt

```diff
@@ -42,7 +42,6 @@ package com.mapbox.navigation.core {
     method public void unregisterTripSessionStateObserver(com.mapbox.navigation.core.trip.session.TripSessionStateObserver tripSessionStateObserver);
     method public void unregisterVoiceInstructionsObserver(com.mapbox.navigation.core.trip.session.VoiceInstructionsObserver voiceInstructionsObserver);
     method public void updateSensorEvent(android.hardware.SensorEvent sensorEvent);
-    method public void useExtendedKalmanFilter(boolean useEKF);
```

Comment:
This is noting that we're breaking SemVer, which is 👌 but we should add the `SEMVER-MAJOR` label 🙏

Reply:
done